### ISSUE TITLE
problem on receiving any email notification from the support site posting

### ISSUE THREAD

**Issue by LihuaJulieZhu** [Bioconductor/support.bioconductor.org#40]
Hi,

For some reasons, after the new release 3.12, I no longer receive any email notification from the support site posting. FYI, I was also not able to receive my own post a while back.
 
Could you please help fixing the issues?
 
Many thanks for your help!
 
Best regards,
 
Julie

**Comment by lshep** [comment]
Do you have email as the notification settings for your account?

**Comment by lshep** [comment]
and what email is associated with your account?

**Comment by LihuaJulieZhu** [comment]
Julie.zhu@umassmed.edu

Thanks!

Best,

Julie

**Comment by LihuaJulieZhu** [comment]
Yes, I never changed the setting since its beginning.

Thanks!

Best,
Julie



****************************************
Lihua Julie Zhu, Ph.D.
Professor
Molecular, Cell and Cancer Biology (MCCB)
Head of Bioinformatics Core, MCCB
Program in Bioinformatics and Integrative Biology
Program in Molecular Medicine
University of Massachusetts Medical School
364 Plantation Street
Worcester, MA
https://profiles.umassmed.edu/display/129880

**Comment by guidohooiveld** [comment]
I experienced the same problem as Julie (TS).
I thus edited my profile to receive emails for both `Digest Preferences` and `Notifications`.

This *seemed* to work, but I now noticed that my email address is one of those explicitly listed in the 'to field' when receiving mail from the support foum. This is also the case for Robert Castelo, James MacDonald, Herve Pages, and Martin Morgan. In other words, we all will get mail if the 'reply to all' is selected by the receiver. This was not the case before the forum update was applied. 

To be honest, I don't like the current implementation. (privacy, spam, etc). Could this behavior be changed/reversed? Thus hide email addresses (like it used to be)?

Thanks,
Guido

**Comment by Natay** [comment]
@guidohooiveld Thanks for letting us know, we will look into this and push out a fix.

**Comment by LihuaJulieZhu** [comment]
Hi Guido, thanks for confirming that this is a general issue and for the workaround!

Best regards,

Julie

**Comment by guidohooiveld** [comment]
@Natay Thanks!
FYI: I noticed that Julie's email address is now also included in the 'to field' send by the support site; likely because she just edited her profile....

**Comment by lgatto** [comment]
Hello - same issue here. I have set my *Digest Preferences* and *Notifications* to email for now.

**Edit:** Hmm, that won't do it I'm afraid, as I'm now getting completely unrelated emails, presumably because of the digest preferences, which I have switched back to *Never*. It does seem that the *Notifications* aren't working.

**Comment by Natay** [comment]
@lgatto the `Digest Preferences` are decoupled from the `Notifications`. 
Sorry for the confusion, we will add more info to the FAQ and docs to better explain this. 

@guidohooiveld the `To` Field no longer contains all other recipients. 

@LihuaJulieZhu , we have fixed your original issue so notifications are correctly sent out for watched tags. To get notification for posts, make sure the __Notifications__ options is set to __Email__.

We have pushed out a fix and will keep an eye out to close this issue once we are sure it runs smooth, thanks for your patience.
